components/question/RangeSlider/index.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+import { buildSurveyQuestionsWithAnswers } from '../Details/buildSurveyQuestionsWithAnswers';
+import styles from './index.module.scss';
+
+export const ACTIVE_SLIDER_BACKGROUND = 'rgba(255, 255, 255)';
+export const INACTIVE_SLIDER_BACKGROUND = 'rgba(255, 255, 255, 0.18)';
+
+export const rangeSliderTestIds = {
+  base: 'range-slider',
+  input: 'range-slider__input',
+};
+
+const RangeSlider = ({ id, answers, setSurveyQuestionsWithAnswers }) => {
+  const [currentValue, setCurrentValue] = useState(-1);
+  const minValue = 0;
+  const maxValue = answers.length - 1;
+  const progress = `${(currentValue / maxValue) * 100}%`;
+
+  useEffect(() => {
+    setCurrentValue(-1);
+  }, [id]);
+
+  const onRangeValueChange = value => {
+    const surveyQuestionsWithAnswers = buildSurveyQuestionsWithAnswers(id, answers[value].id);
+    setCurrentValue(value);
+    setSurveyQuestionsWithAnswers(prevSurveyQuestionWithAnswer => {
+      let newSurveyQuestionWithAnswer = prevSurveyQuestionWithAnswer.slice();
+      newSurveyQuestionWithAnswer = newSurveyQuestionWithAnswer.filter(response => response.id !== id);
+
+      return [...newSurveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+    });
+  };
+
+  const inputStyle = {
+    background: `linear-gradient(to right, ${ACTIVE_SLIDER_BACKGROUND} 0% ${progress}, ${INACTIVE_SLIDER_BACKGROUND} ${progress} 100%)`,
+  };
+
+  return (
+    <div className="h-14 flex items-center mt-16" data-test-id={rangeSliderTestIds.base}>
```

Comment:
Should we move this CSS into `./index.module.scss` as well?

Reply:
I think it's a very good idea, I removed the wrapper div and moved these classes onto `./index.module.scss` in 3a7ab36a48d3b13c8c1c263372d878d4ed669872, Thank you!